cypress/integration/resmokeLogs/resmoke_logView.ts

```diff
@@ -34,6 +34,9 @@ describe("Basic resmoke log view", () => {
     );
     cy.get(".ReactVirtualized__Grid").scrollTo("right");
   });
+  it("log header should not show breadcrumbs since the task is not found by Evergreen", () => {
```

Comment:
Sorry I didn't clarify this earlier. You need to use the logkeeperBuildMetadata resolver to get this data. It can probably be updated to return a full task instead of just the taskId 
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/4605522/229868155-40915025-5789-4443-98b6-52125a5c6dc1.png">

Reply:
I can put up a quick pr to surface this data. 
```graphql
{
  logkeeperBuildMetadata(buildId: "7e208050e166b1a9025c817b67eee48d") {
    task {
      displayName
      patchNumber
      status
      versionMetadata {
        id
        isPatch
        projectIdentifier
        message
        revision
      }
    }
  }
}
```

Reply:
Done!